controllers/vspherevm_controller.go

```diff
@@ -348,6 +348,11 @@ func (r vmReconciler) reconcileNormal(ctx *context.VMContext) (reconcile.Result,
 	// Update the VSphereVM's network status.
 	r.reconcileNetwork(ctx, vm)
```

Comment:
It looks like this call [returns an error](https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/master/controllers/vspheremachine_controller.go#L518-L521) when address isn't found. can probably just use that?